install/installer/pkg/common/storage.go

```diff
@@ -46,16 +46,11 @@ func StorageConfig(context *RenderContext) storageconfig.StorageConfig {
 
 	if context.Config.ObjectStorage.S3 != nil {
 		res = &storageconfig.StorageConfig{
-			Kind: storageconfig.MinIOStorage,
-			MinIOConfig: storageconfig.MinIOConfig{
-				Endpoint:            context.Config.ObjectStorage.S3.Endpoint,
-				AccessKeyIdFile:     filepath.Join(storageMount, "accessKeyId"),
-				SecretAccessKeyFile: filepath.Join(storageMount, "secretAccessKey"),
-				Secure:              !context.Config.ObjectStorage.S3.AllowInsecureConnection,
-				Region:              context.Config.Metadata.Region,
-				ParallelUpload:      100,
-
-				BucketName: context.Config.ObjectStorage.S3.BucketName,
```

Comment:
May I say we no longer support MinIO? Or do we no longer support insecure S3?

Reply:
See Simon's [comment](https://github.com/gitpod-io/gitpod/pull/14970#issuecomment-1344202161)

Reply:
Is preview env uses MinIO?

Reply:
I believe preview envs use the in-cluster Minio, which is not affected by this change. This is just the struct which is how the storage ingested the configuration - for an S3 deployment, there is no Minio container deployed.

For reference, the in-cluster config is [here](https://github.com/gitpod-io/gitpod/blob/76244c76794d6d8d965bf17be7ad7e38e1ccf569/install/installer/pkg/common/storage.go#L56,L68)

Reply:
+1 to what Simon said. Preview environments use the in cluster config which is not affected by this change.

Reply:
I got it. Thank you @mrsimonemms and @Furisto.